react/features/base/config/reducer.js

```diff
@@ -18,15 +18,7 @@ const INITIAL_STATE = {
     // API, through the React Native bridge and eventually to JavaScript.
     // Because the audio levels are of no interest to the mobile app, it is
     // fastest to merely disable them.
-    disableAudioLevels: true,
-
-    // FIXME Lib-jitsi-meet uses HTML script elements to asynchronously load
-    // certain pieces of JavaScript. Unfortunately, the technique doesn't work
-    // on React Native (because there are no HTML elements in the first place).
-    // Fortunately, these pieces of JavaScript currently involve third parties
-    // and we can temporarily disable them (until we implement an alternative to
-    // async script elements on React Native).
-    disableThirdPartyRequests: true
```

Comment:
It still lives here: https://github.com/saghul/lib-jitsi-meet/blob/892d318f1dee96b772796b1f0c290e01d52ad1f9/modules/statistics/statistics.js#L106